pyproject.toml

```diff
@@ -8,7 +8,7 @@ authors = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.11"
+python = "~3.11"
 django = "^5.1"
 whitenoise = "^6.8.2"
 drunc = { git = "https://github.com/DUNE-DAQ/drunc.git", rev = "f068eaccb840da03a104adba2da3f27454c218d5" }
```

Comment:
Nothing to do with this PR, but I've seen that [v0.11.1](https://github.com/DUNE-DAQ/drunc/releases/tag/v0.11.1) was released yesterday.